package com.example.mainactivity;

import android.os.Bundle;
import android.text.Layout;
import android.widget.Button;
import android.widget.TextView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

import java.util.Random;

public class MainActivity extends AppCompatActivity {


    int contador = 0;
    Button b;
    TextView tv;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_main);

        tv = findViewById(R.id.tv);
        b = findViewById(R.id.button);
        b.setOnClickListener(v -> {
            contador++;
            Random random=new Random();
            int r = random.nextInt( 100);
            tv.setText(Integer.toString(r));

        });
    }
}